fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+swagger: '2.0'
+  
+info:
+  version: 1.0.0 
+  title: Draft Orders API
+  x-logo:
+    url: "https://andrewjtech123.github.io/imagehost/shopify_logo_black.png"
+    backgroundColor:
+  
+  description: >
+    # Draft Orders API 
```

Comment:
looks like we're getting a double title:

![screen shot 2016-11-28 at 8 49 18 am](https://cloud.githubusercontent.com/assets/10617892/20670603/9a6b7400-b547-11e6-946a-a090a28da003.png)

Perhaps omit the H1 at line 11?